src/sentry/services/http.py

```diff
@@ -182,6 +191,7 @@ def run(self):
             host, port = os.environ["UWSGI_HTTP_SOCKET"].split(":")
             httpd = make_server(host, int(port), application)
             httpd.serve_forever()
+            raise AssertionError("unreachable")
```

Comment:
this is to keep `-> NoReturn` here -- technically `serve_forever` _could_ terminate but we don't subclass the server and call `.shutdown()`